tools/x-docs/src/docs/guides/components/setup.md

```diff
@@ -0,0 +1,105 @@
+## Working on components
+
+You've been through the [quick start guide](/tools/x-docs/src/docs/guides/components/getting-started.md), and you want to make your first change to x-dash. This guide will take you through the process for working on x-dash components, demoing and testing your changes, and getting them merged and released.
+
+## Component development process
+
+Open the monorepo in your editor and [have a look around](#getting-around-the-monorepo). While the development server is running, whenever you make a change to a component, it will update in the component explorer without a reload. This way, you can quickly verify your changes in the browser.
```

Comment:
"component explorer" = workbench?

Reply:
yeah. i'm not 100% happy with the terminology here. we should make it consistent everywhere. which might just mean calling it storybook